packages/components/src/workers/parquetWorkerClient.ts

```diff
@@ -24,7 +25,7 @@ const pending = new Map<number, QueryAgent>()
 
 function getWorker() {
   if (!worker) {
-    worker = new Worker(new URL('worker.min.js', import.meta.url))
+    worker = new ParquetWorker() //new URL('./parquetWorker', import.meta.url), { type: 'module' })
```

Comment:
nit: remove commented out code

Aside: we probably can't assume the location of the worker bundle, since projects depending on it might have a different path structure. Not sure exactly how but we probably need it to be configurable.

Reply:
I understand we have two options:
1. give explanations to users to include the worker in their bundler (copy the worker files, that are bundled in the lib's dist/assets/ directory, to the same directory in their app)
2. compile the worker as a base64 string.

I'll review both of them.

Reply:
Related discussion: https://github.com/vitejs/vite/issues/7352#issuecomment-1312236119. It's not clear what is the way to go.

Reply:
https://github.com/domoran/vite_worker_lib_example is interesting:

- the lib build creates the worker file in assets/, as we do
- the consumer builder (vite) is able to copy the asset by default

If we use vite for the CLI, it should be working without much config. Though, we might have to write something in the README for users that will not rely on vite for their app...

<details><summary>See the build details (bundle structure and HTML contents)</summary>
<pre><code>
➜  vite_worker_lib_example git:(master) ls
consumer  lib  README.md   
➜  vite_worker_lib_example git:(master) cd lib  
➜  lib git:(master) ✗ npm run build                                                         
                                                                                            
> lib@0.0.0 build
> tsc && vite build
                                              
vite v5.2.8 building for production...
✓ 2 modules transformed.                                                                    
dist/assets/webworker-C7uLr2oP.js  0.25 kB   
dist/lib.js                        0.46 kB │ gzip: 0.27 kB
dist/lib.umd.cjs  0.94 kB │ gzip: 0.46 kB
✓ built in 76ms
➜  lib git:(master) ✗ cd ../consumer 
➜  consumer git:(master) ✗ npm run build

> consumer@0.0.0 build
> tsc && vite build

vite v5.2.8 building for production...
✓ 5 modules transformed.
dist/assets/webworker-C7uLr2oP.js  0.25 kB
dist/index.html                    0.38 kB │ gzip: 0.27 kB
dist/assets/index-B1ZSCV_w.js      1.67 kB │ gzip: 0.69 kB
✓ built in 110ms
➜  consumer git:(master) ✗ cat dist/index.html 
&#60;!doctype html&#62;
&#60;html lang="en"&#62;
  &#60;head&#62;
    &#60;meta charset="UTF-8" /&#62;
    &#60;link rel="icon" type="image/svg+xml" href="/vite.svg" /&#62;
    &#60;meta name="viewport" content="width=device-width, initial-scale=1.0" /&#62;
    &#60;title&#62;Vite + TS&#60;/title&#62;
    &#60;script type="module" crossorigin src="/assets/index-B1ZSCV_w.js"&#62;&#60;/script&#62;
  &#60;/head&#62;
  &#60;body&#62;
    &#60;div id="app"&#62;&#60;/div&#62;
  &#60;/body&#62;
&#60;/html&#62;
</code></pre>
</details>

Reply:
Well, I ended inlining the worker (hence also the hyparquet and hyparquet-compressor libs) in the build lib js... The size is now:
```
dist/main.es.js  234.34 kB │ gzip: 123.03 kB │ map: 119.65 kB
dist/main.umd.js  223.48 kB │ gzip: 121.78 kB │ map: 116.58 kB
```

If it's an issue, we can instead switch to the other solution by reverting https://github.com/hyparam/hyperparam-cli/pull/14/commits/a755dfd9ee45f93e678af870796b9e40791bf6d3. The client app will have to take care of copying the build worker file to its assets (might be automatic if using Vitge, see above comment)